Dockerfile

```diff
@@ -13,6 +13,6 @@ RUN pip install "poetry==$POETRY_VERSION" "uwsgi==$UWSGI_PIP_VERSION" "psycopg2=
 RUN poetry self add "poetry-dynamic-versioning[plugin]"
 ADD . /app
 RUN poetry install --no-dev
-RUN python tools/install_bootstrap.py -u
+RUN python tools/install_bootstrap.py -u # FIXME
```

Comment:
@frcroth django management command do a full setup before running (to make interacting with the database etc. easier). When building the container, we cannot do that because the environment to run django in is not there yet. This means we have to either revert my changes and keep the bootstrap installation as a standalone script or handle the bootstrap installation when deploying. What are your thoughts on this?

Reply:
I would prefer the bootstrap stuff to remain in tools in that case. I think it still makes sense there, because I would expect django management tools to interact with django and this tool doesn't really do that.

Reply:
@frcroth fixed